modules/sd_models.py

```diff
@@ -194,27 +194,17 @@ def select_checkpoint():
 }
 
 
-def transform_checkpoint_dict_key(k):
-    for text, replacement in checkpoint_dict_replacements.items():
-        if k.startswith(text):
-            k = replacement + k[len(text):]
-
-    return k
-
-
 def get_state_dict_from_checkpoint(pl_sd):
     pl_sd = pl_sd.pop("state_dict", pl_sd)
     pl_sd.pop("state_dict", None)
 
-    sd = {}
-    for k, v in pl_sd.items():
-        new_key = transform_checkpoint_dict_key(k)
-
-        if new_key is not None:
-            sd[new_key] = v
-
-    pl_sd.clear()
-    pl_sd.update(sd)
+    state_dict_keys = pl_sd.keys()
+    for key in state_dict_keys:

```

Comment:
It's generally not safe to mutate a dict you're iterating over (and `keys` is a live view), so grab a copy of the keys as a list:

```suggestion
    for key in list(pl_sd):
```